frontend/src/store/modules/auth.js

```diff
@@ -14,19 +14,22 @@ const getters = {
   },
   hasPermission: (state) => (permission) => {
     return state.permissions.includes(permission)
-  }
+  },
+  hasAnyPermission: (state) => {
```

Comment:
Personal preference but I would tend to use hasPermissions

Reply:
Wanted to make it easily distinguishable from hasPermission so might leave as is.